adam/curriculum/phase1_curriculum.py

```diff
@@ -163,15 +163,15 @@ def build_object_multiples_situations(
     constraining_relations=[bigger_than(_PERSON_0, _INANIMATE_OBJECT_0)],
 )
 
-# PERSON_HAS_OBJECT_SUB_CURRICULUM = _phase1_instances(
-#     "person has object",
-#     situations=sampled(
-#         PERSON_HAS_OBJECT_TEMPLATE,
-#         chooser=_CHOOSER,
-#         ontology=GAILA_PHASE_1_ONTOLOGY,
-#         max_to_sample=100,
-#     ),
-# )
+PERSON_HAS_OBJECT_SUB_CURRICULUM = _phase1_instances(
```

Comment:
Don't we need a property restriction on the object variable for `PERSON_HAS_OBJECT_TEMPLATE`?